Fix:

maintests.c:232:3: error: implicit declaration of function 'setup_log_handler' is invalid in C99 [-Werror,-Wimplicit-function-declaration]

utimer.c:164:3: error: implicit declaration of function 'g_type_init' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
diff -ru configure configure
--- src/log.c.orig	2009-05-30 16:51:30.000000000 -0500
+++ src/log.c	2020-11-20 18:41:08.000000000 -0600
@@ -71,7 +71,7 @@
 
 
 /* set up the verbose/debug log handler */
-void setup_log_handler ()
+void setup_log_handler (void)
 {
   g_log_set_handler (NULL,
                      G_LOG_LEVEL_INFO
diff -ru src/log.h src/log.h
--- src/log.h.orig	2009-05-30 11:53:25.000000000 -0500
+++ src/log.h	2020-11-20 18:41:08.000000000 -0600
@@ -27,6 +27,7 @@
 #define g_info(format...) g_log(G_LOG_DOMAIN, G_LOG_LEVEL_INFO, format)
 #endif 
 
+void setup_log_handler (void);
 
 
 #endif /* LOG_H */
diff -ru src/tests/maintests.c src/tests/maintests.c
--- src/tests/maintests.c.orig	2009-06-01 09:31:26.000000000 -0500
+++ src/tests/maintests.c	2020-11-20 18:41:37.000000000 -0600
@@ -30,6 +30,7 @@
 #endif 
 
 #include <glib-object.h>
+#include "../log.h"
 #include "../utimer.h"
 
 #define TEST_DURATION_MAX_OFFSET_MSECONDS 100
diff -ru src/utimer.c src/utimer.c
--- src/utimer.c.orig	2009-06-01 09:23:45.000000000 -0500
+++ src/utimer.c	2020-11-20 18:43:17.000000000 -0600
@@ -21,6 +21,7 @@
  * 
  */
 
+#include <glib-object.h>
 #include "utimer.h"
 
 //~ static    char            **remaining_args;
